crates/figma_import/src/transform_flexbox.rs

```diff
@@ -1325,29 +1330,47 @@ fn visit_node(
                 } else {
                     None
                 };
-                runs.push(StyledTextRun {
-                    text: current_run.clone(),
-                    style: TextStyle {
-                        text_color,
-                        font_size,
-                        font_family: sub_style.font_family.clone(),
-                        font_weight,
-                        font_style: font_style, // Italic or Normal
-                        font_stretch: style.node_style.font_stretch, // Not in SubTypeStyle.
-                        letter_spacing: sub_style
-                            .letter_spacing
-                            .unwrap_or(style.node_style.letter_spacing.unwrap_or(0.0)),
-                        text_decoration: text_decoration,
-                        line_height: style.node_style.line_height,
-                        font_features: convert_opentype_flags(
-                            &sub_style
-                                .opentype_flags
-                                .clone()
-                                .unwrap_or(text_style.opentype_flags.clone()),
-                        ),
-                        hyperlink,
-                    },
-                });
+                let style = TextStyle {
+                    text_color,
+                    font_size,
+                    font_family,
+                    font_weight,
+                    font_style,                                  // Italic or Normal
+                    font_stretch: style.node_style.font_stretch, // Not in SubTypeStyle.
+                    letter_spacing: sub_style
+                        .letter_spacing
+                        .unwrap_or(style.node_style.letter_spacing.unwrap_or(0.0)),
+                    text_decoration,
+                    line_height: style.node_style.line_height,
+                    font_features: convert_opentype_flags(
+                        &sub_style
+                            .opentype_flags
+                            .clone()
+                            .unwrap_or(text_style.opentype_flags.clone()),
+                    ),
+                    hyperlink,
+                };
+                let mut has_handled = false;
+                if runs.len() > 0 {
+                    let last_run = runs.get(runs.len() - 1);
+                    if last_run.is_some() {
+                        let last_run_style = last_run.unwrap().style.clone();
+                        if last_run_style == style {
+                            error!("the two styles are the same.");
+                            let to_be_merged = runs.pop();
+                            let mut new_run = String::new();
+                            if to_be_merged.is_some() {
```

Comment:
Normally we try to avoid `unwrap()`, so we'd do something like `if let Some(tbm) = to_be_merged { push(&tbm.text); }`

Reply:
Done.